packages/stellar-axelar-std-derive/src/contractstorage.rs

```diff
@@ -0,0 +1,177 @@
+    let variant_name = &variant.ident;
+
+    match &variant.fields {
+        Fields::Unit => {
+            quote! {
+                #variant_name
+            }
+        }
+        Fields::Named(FieldsNamed { named, .. }) => {
+            let types = named.iter().map(|f| &f.ty);
+            quote! {
+                #variant_name(#(#types),*)
+            }
+        }
+        _ => panic!("Only unit variants or named fields are supported in storage enums."),
+    }
+}
+
+fn storage_attributes(attrs: &[Attribute]) -> StorageAttributes {
+    let mut storage_attributes = StorageAttributes::default();
+
+    for attr in attrs {
+        let path_str = attr.path().to_token_stream().to_string();
+
+        match path_str.as_str() {
+            "instance" => {
+                storage_attributes.storage_type = StorageType::Instance;
+            }
+            "persistent" => {
+                storage_attributes.storage_type = StorageType::Persistent;
+            }
+            "temporary" => {
+                storage_attributes.storage_type = StorageType::Temporary;
+            }
+            "value" => {
+                if let Meta::List(list) = &attr.meta {
+                    if let Ok(ty) = list.parse_args::<Type>() {
+                        storage_attributes.value_type = Some(ty);
+                    }
+                }
+            }
+            _ => {}
+        }
+    }
+
+    storage_attributes
+}
+
+fn storage_fns(
+    enum_name: &Ident,
+    variant: &Variant,
+    storage_attrs: &StorageAttributes,
+) -> TokenStream {
+    let variant_ident = &variant.ident;
+
+    let (field_names, field_types) = match &variant.fields {
+        Fields::Unit => (vec![], vec![]),
+        Fields::Named(fields) => {
+            let names = fields.named.iter().map(|f| &f.ident).collect();
+            let types = fields.named.iter().map(|f| &f.ty).collect();
+            (names, types)
+        }
+        _ => panic!("Only unit variants or named fields are supported in storage enums."),
+    };
+
+    let value_type = storage_attrs
+        .value_type
+        .as_ref()
+        .expect("value type required");
+
+    let getter_name = format_ident!("get_{}", variant_ident.to_string().to_snake_case());
+    let setter_name = format_ident!("set_{}", variant_ident.to_string().to_snake_case());
+
+    let storage_method = match storage_attrs.storage_type {
+        StorageType::Instance => quote! { instance },
+        StorageType::Persistent => quote! { persistent },
+        StorageType::Temporary => quote! { temporary },
+    };
+
+    let ttl_fn = match storage_attrs.storage_type {
+        StorageType::Persistent => quote! {
+            use stellar_axelar_std::ttl::extend_persistent_ttl;
+            extend_persistent_ttl(env, &key);
+        },
+        StorageType::Instance => quote! {
+            use stellar_axelar_std::ttl::extend_instance_ttl;
+            extend_instance_ttl(env);
```

Comment:
```suggestion
            stellar_axelar_std::ttl::extend_instance_ttl(env);
```
no need to use `use` unless a trait needs to be imported